pkgs/by-name/pu/pulsemeeter/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  libpulseaudio,
+  libappindicator,
+  gobject-introspection,
+  gtk3,
+
+}:
+python3Packages.buildPythonApplication rec {
```

```suggestion
python3Packages.buildPythonApplication (finalAttrs: {
```

`finalAttrs` is usually preferred here over rec to allow overriding. See [the manual](https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes)